binzx/otomi

```diff
@@ -43,6 +43,7 @@ silent() {
 }
 
 docker_tag_exists() {
+  [ -n "$CI" ] && return 0
```

Comment:
why is this added?

Reply:
To skip when running in CI mode

Reply:
is that needed? and here in this function? out of place and confusing

Reply:
please create conditions where they apply, so it stays maintainable